ExchangeSharp/API/Exchanges/Coinbase/ExchangeCoinbaseAPI.cs

```diff
@@ -351,6 +351,55 @@ private ExchangeTicker ParseTickerWebSocket(JToken token)
             };
         }
 
+        protected override IWebSocket OnGetTradesWebSocket(Action<KeyValuePair<string, ExchangeTrade>> callback, params string[] symbols)
+        {
+            if (callback == null)
+            {
+                return null;
+            }
+
+            return ConnectWebSocket("/", (_socket, msg) =>
+            {
+                JToken token = JToken.Parse(msg.ToStringFromUTF8());
```

Comment:
Same as above. Consider deserializing to a predefined object for perf considerations